aineko/core/dataset.py

```diff
@@ -51,12 +56,14 @@ class DatasetConsumer:
         node_name: name of the node that is consuming the dataset
         pipeline_name: name of the pipeline
         dataset_config: dataset config
-        broker: broker to connect to (ip and port: "54.88.142.21:9092")
+        bootstrap_servers: bootstrap_servers to connect to (e.g. "1.2.3.4:9092")
         prefix: prefix for topic name (<prefix>.<dataset_name>)
         has_pipeline_prefix: whether the dataset name has pipeline name prefix
 
     Attributes:
         consumer: Kafka consumer object
+        cached: if the high watermark offset has been cached
+        (updated when message consumed)
```

Comment:
```suggestion
            (updated when message consumed)
```
Args with a multi-line description should have a hanging indent.